packages/neon/neon/lib/src/widgets/nextcloud_logo.dart

```diff
@@ -12,5 +13,6 @@ class NextcloudLogo extends StatelessWidget {
         package: 'neon',
         width: 100,
         height: 100,
+        semanticsLabel: AppLocalizations.of(context).nextcloudLogo,
```

Comment:
I think the normal nextcloud brand name `Nextcloud` would be enough.

My Idea was to have something like "Neon \n works with \n Nextcloud" while we now get "Works with \n Nextcloud logo".

(The neon logo should also get a semantic node in a followup)

Reply:
No the logo on the login page is wrapped with extra semantics exactly because of this.